java-frontend/src/main/java/org/sonar/java/AbstractJavaClasspath.java

```diff
@@ -99,13 +100,19 @@ private boolean hasJavaSources() {
     return fs.hasFiles(fs.predicates().and(fs.predicates().hasLanguage("java"), fs.predicates().hasType(fileType)));
   }
 
-  private static Set<File> getFilesForPattern(Path baseDir, String pathPattern, boolean libraryProperty) {
+  private Set<File> getFilesForPattern(Path baseDir, String pathPattern, boolean libraryProperty) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this method 6, down to the maximum allowed 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1142)